### PR TITLE
Suggestions by NLnet

### PR DESCRIPTION
**Pull request by ghost**
- [x] menu: link to forum
- [x] NGIzero attribution

### CODE REVIEW

**Comment by pabzm** [comment]
Can we not use a `.box` for this, please? We have one of those elements at the top of each page, and no where else. It breaks the visual concept to repeat it.

Maybe use `.notification` or some custom styling?

Otherwise I agree.

**Review by pabzm — CHANGES_REQUESTED** [review]
Can we not use a .box for this, please? We have one of those elements at the top of each page, and no where else. It breaks the visual concept to repeat it.

Maybe use .notification or some custom styling?

Otherwise I agree.

**Comment by hpk42** [comment]
looks good to me -- suggest the menu entry to be "Examples / Forum" so to have examples in the name indeed.

**Review by hpk42 — APPROVED** [review]

**Comment by missytake** [comment]
Hm, "Examples / Forum" didn't look very good in the layout - that they can also use the forum to ask questions is hopefully coming intuitively when others use the forum that way?

**Comment by hpk42** [comment]
sorry didn't realize it before -- it's a bit much text -- i'd strike the ", Content and Technology under grant agreement No 825310. Applications are still open, you can apply today." part completely.

File: index.md [hunk 1]
```diff
@@ -22,3 +22,14 @@ The bot doesn't even have to be connected all the time. It obviously can't read
 Read on:
 * [How to run a Delta Chat bot]({{ site.baseurl }}howto.html)
 * [Developer information]({{ site.baseurl }}development.html)
+
+This project was funded through the [NGI0 PET](https://nlnet.nl/PET) Fund, a
+fund established by [NLnet](https://nlnet) with financial support from the
+European Commission's [Next Generation Internet](https://ngi.eu/) programme,
+under the aegis of DG Communications Networks, Content and Technology under
+grant agreement No 825310. **Applications are still open, you can [apply
+today](https://nlnet.nl/propose).**
```
Comment (hpk42) [inline index.md]:
```suggestion
Delta Chat Bot developments were partially funded through the [NGI0 NLNET PET](https://nlnet.nl/PET) Fund, with financial support from the
European Commission's [Next Generation Internet](https://ngi.eu/) programme.**
```

**Review by pabzm — APPROVED** [review]
I'm not very happy with "Example bots" linking to the forum, but that's a minor thing I can live with for now. Maybe we will find a better wording later.

**Pull request merged**